report_templates/ansible_inventory.erb

```diff
@@ -0,0 +1,137 @@
+<%- opts = content_attrs ? { includes: [:content_facet] } : {} -%>
+<%- opts[:includes] += [:organization, :location, :hostgroup, :host_collections, :interfaces] %>
+<%- load_hosts(opts).each_record do |host| -%>
+<%-   content_facet = host_content_facet(host) if content_attrs -%>
+<%-   content_attribute_data = nil -%>
+<%-   if content_facet -%>
+<%-     content_attribute_data = {
+          'applicable_module_stream_count': content_facet.applicable_module_stream_count,
+          'applicable_package_count': content_facet.applicable_rpm_count,
+          'content_source': content_facet.content_source,
+          'content_source_id': content_facet.content_source_id,
+          'content_source_name': content_facet.content_source_name,
+          'content_view': {
+              'id': content_facet.content_view_id,
+              'name': content_facet.content_view_name,
+          },
+          'content_view_id': content_facet.content_view_id,
+          'content_view_name': content_facet.content_view_name,
+          'errata_counts': content_facet.errata_counts,
+          'id': content_facet.id,
+          'kickstart_repository': content_facet.kickstart_repository,
+          'kickstart_repository_id': content_facet.kickstart_repository_id,
+          'kickstart_repository_name': content_facet.kickstart_repository_name,
+          'lifecycle_environment': {
+              'id': content_facet.lifecycle_environment_id,
+              'name': content_facet.lifecycle_environment_name,
+          },
+          'lifecycle_environment_id': content_facet.lifecycle_environment_id,
+          'lifecycle_environment_name': content_facet.lifecycle_environment_name,
+          'upgradable_module_stream_count': content_facet.upgradable_module_stream_count,
+          'upgradable_package_count': content_facet.upgradable_rpm_count,
+          'uuid': content_facet.uuid,
+}
+-%>
+<%-   end -%>
+<%-   inventory_data = { 'Name': host.name }
+      inventory_data.update('Organization': host.organization) if input('Organization') == 'yes'
+      inventory_data.update('Location': host.location) if input('Location') == 'yes'
+      hostgroup = host.hostgroup.title if host.hostgroup
+      inventory_data.update('Host Group': hostgroup) if input('Host Group') == 'yes'
+      host_collections = host.host_collections.map { |c| c.name } if host.host_collections
+      inventory_data.update('Host Collections': host_collections) if input('Host Collections') == 'yes'
+      inventory_data.update('IPv4': host.ip) if input('IPv4') == 'yes'
+      inventory_data.update('Subnet': host.subnet) if input('Subnet') == 'yes'
+      smart_proxies = host.smart_proxies.map { |p| p.name } if host.smart_proxies
+      inventory_data.update('Smart Proxies': smart_proxies) if input('Smart Proxies') == 'yes'
+      inventory_data.update('Content Attributes': content_attribute_data) if content_attrs
+      inventory_data.update('Facts': host.facts) if input('Facts') == 'yes'
```

Comment:
We would ideally want these keys in lowercase.

Reply:
Done.